slothy/core/slothy.py

```diff
@@ -393,14 +392,14 @@ def fusion_loop(self, loop_lbl):
 
         assert SourceLine.is_source(self.source)
 
-    def optimize_loop(self, loop_lbl, postamble_label=None):
+    def optimize_loop(self, loop_lbl, postamble_label=None, cnt_iter_factor=1):
```

Comment:
Maybe add a word of documentation for `cnt_iter_factor`? And, while at it, for `postamble_label`? :)